tuner/tuner/dispatch_constraints.py

```diff
@@ -134,6 +141,35 @@ def generate_constraints(
     return constraints
 
 
+def getMMAAttr(
+    output_type: ir.IntegerType | ir.FloatType,
+    m: int,
+    n: int,
+    k: int,
+    lhs_type: ir.IntegerType | ir.FloatType,
+    rhs_type: ir.IntegerType | ir.FloatType,
+) -> iree_gpu.MMAAttr:
+    for mma_intrinsic in iree_gpu.MMAIntrinsic:
+        mma_attr = iree_gpu.MMAIntrinsicAttr.get(mma_intrinsic).mma
```

Comment:
Here we can construct mma directly from the intrinsic without creating an mmaintrinsicattr first